backend/internal/storage/postgres/storage.go

```diff
@@ -40,6 +41,7 @@ func ConnectDatabase(host, user, password, dbname, port string) *pgxpool.Pool {
 
 func NewRepository(db *pgxpool.Pool) *storage.Repository {
 	return &storage.Repository{
-		User: user.NewUserRepository(db),
+		User:   user.NewUserRepository(db),
+		Review: review.NewReviewRepository(db),
```

Comment:
We can use the same import here for the NewReviewRepository(db). Honestly might be better to change user into something more universal.